$OpenBSD: patch-tmac_groff_mdoc_man,v 1.5 2013/11/11 06:41:02 schwarze Exp $
chunk 1: More OpenBSD versions (to be submitted upstream).
chunks 2-4: Implement %C (accepted and committed upstream).
--- tmac/groff_mdoc.man.orig	Thu Feb  7 13:06:08 2013
+++ tmac/groff_mdoc.man	Wed Jul 24 23:31:38 2013
@@ -883,7 +883,7 @@ the release ID.
 .It OpenBSD
 2.0, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 2.8, 2.9, 3.0, 3.1, 3.2, 3.3, 3.4,
 3.5, 3.6, 3.7, 3.8, 3.9, 4.0, 4.1, 4.2, 4.3, 4.4, 4.5, 4.6, 4.7, 4.8, 4.9,
-5.0
+5.0, 5.1, 5.2, 5.3, 5.4
 .It DragonFly
 1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.8, 1.8.1, 1.10, 1.12, 1.12.2, 2.0, 2.2,
 2.4, 2.6, 2.8, 2.9, 2.9.1, 2.10, 2.10.1, 2.11
@@ -2637,7 +2637,7 @@ Reference author name; one name per invocation.
 .It Li .%B
 Book title.
 .It Li .%C
-City/place (not implemented yet).
+City/place.
 .It Li .%D
 Date.
 .It Li .%I
@@ -2684,7 +2684,8 @@ Example:
 \&.%A "John Foo"
 \&.%T "Implementation Notes on foobar(1)"
 \&.%R "Technical Report ABC\-DE\-12\-345"
-\&.%Q "Drofnats College, Nowhere"
+\&.%Q "Drofnats College"
+\&.%C "Nowhere"
 \&.%D "April 1991"
 \&.Re
 .Ed
@@ -2697,7 +2698,8 @@ produces
 .%A "John Foo"
 .%T "Implementation Notes on foobar(1)"
 .%R "Technical Report ABC-DE-12-345"
-.%Q "Drofnats College, Nowhere"
+.%Q "Drofnats College"
+.%C "Nowhere"
 .%D "April 1991"
 .Re
 .Ed
